roles/vdm/tasks/postgres/postgres-instance.yaml

```diff
@@ -145,6 +145,15 @@
 
 - name: postgres instance - external post 2022.10
```

Comment:
Name seems a bit off now given we also have 2022.12 in the mix.

Reply:
@thpang, would you suggest adding 2022.12 into the mix or just removing cad ver from comment entirely?